packages/App/src/store/saga/walletItemsRequest.tsx

```diff
@@ -33,6 +33,15 @@ export function* getWalletItems(): Generator {
     )) as Array<WalletInfo>;
 
     if (response) {
+      response.push( {
```

Comment:
It'll be better not to have this logic in sagas. Sagas and redux are for data only. This special item is part of UI logic, so it should be resolved in UI components, not here